src/tests/ftest/util/soak_utils.py

```diff
@@ -803,15 +702,13 @@ def start_dfuse(self, pool, container, name=None, job_spec=None):
     dfuse.bind_cores = self.params.get("cores", dfuse.namespace, None)
     dfuse.get_params(self)
     # update dfuse params; mountpoint for each container
-    unique = get_random_string(5, self.used)
-    self.used.append(unique)
-    mount_dir = dfuse.mount_dir.value + unique
+    mount_dir = dfuse.mount_dir.value
     dfuse.update_params(mount_dir=mount_dir, pool=pool.identifier, cont=container.identifier)
-    dfuse_log = os.path.join(
-        self.soaktest_dir,
+    dfuselog = os.path.join(
+        self.sharedsoaktest_dir,
         self.test_name + "_" + name + "_`hostname -s`_"
-        "" + "${SLURM_JOB_ID}_" + "daos_dfuse_" + unique)
-    dfuse_env = f"export D_LOG_MASK=ERR;export D_LOG_FILE={dfuse_log}"
+        "" + "${SLURM_JOB_ID}_" + "daos_dfuse.log")
+    dfuse_env = f"export D_LOG_FILE_APPEND_PID=1;export D_LOG_MASK=ERR;export D_LOG_FILE={dfuselog}"
```

Comment:
Not sure if it helps, but FYI that this should work
```
dfuse.env.update({
    'D_LOG_FILE_APPEND_PID': '1',
    'D_LOG_MASK': 'ERR',
    'D_LOG_FILE': dfuselog
})
dfuse_env = dfuse.env.to_export_str()
```

The benefit is that `dfuse.get_params` would get env from here in the yaml as well:
```
dfuse:
    env_vars:
    - foo=bar
```